payjoin-directory/src/lib.rs

```diff
@@ -267,6 +268,59 @@ async fn health_check() -> Result<Response<BoxBody<Bytes, hyper::Error>>, Handle
     Ok(Response::new(empty()))
 }
 
+async fn handle_directory_home_path() -> Result<Response<BoxBody<Bytes, hyper::Error>>, HandlerError>
+{
+    let mut res = Response::new(empty());
+    *res.status_mut() = StatusCode::OK;
+    res.headers_mut().insert(CONTENT_TYPE, HeaderValue::from_static("text/html"));
+
+    let html = r#"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Payjoin Directory</title>
+    <style>
+        body {
+            background-color: #0f0f0f;
+            color: #eaeaea;
+            font-family: 'Courier New', Courier, monospace;
+            padding: 2rem;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        .container {
+            background: #1a1a1a;
+            border: 1px solid #333;
+            border-radius: 8px;
+            padding: 2rem;
+            box-shadow: 0 0 10px rgba(0, 170, 255, 0.2);
+            text-align: center;
+        }
+        h1 {
+            color: #00aaff;
+            margin-bottom: 1rem;
+        }
+        p {
+            color: #ccc;
```

Comment:
mixing html with code is bad practice , However since this is just a minimal page i think those rules can be relaxed here